src/main.rs

```diff
@@ -49,7 +49,20 @@ fn main() {
             .arg(Arg::with_name("target").long("target").short("t").required(false)
                  .default_value("fuzzer_script_1")
                  .help("name of the first fuzz target to create")))
-        .subcommand(SubCommand::with_name("run").about("Run the fuzz target in fuzz/fuzzers")
+        .subcommand(SubCommand::with_name("run").long_about("Run the fuzz target in fuzz/fuzzers")
+            .about(
+"
+
+Run the fuzzer on a given target. Example usage:
```

Comment:
Seems like a bug in clap then? 

Also, you’ve manually split lines and those will look bad on terminals with column count less than the one you used to split at.

Both of these things should either have options in clap or it would be a good idea to submit a feature request. If there’s nothing like that in there, feel free to bors r+ yourself; I’ll fill the bugs with clap at that point.